src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -287,6 +296,49 @@ export class NetCoreTaskHelper implements TaskHelper {
 
         return volumes;
     }
+
+    private async buildWithDotnetSdk(context: DockerBuildTaskContext, buildDefinition: DockerBuildTaskDefinition): Promise<void> {
+        //reference: https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container
+        const publishFlag = NetCoreTaskHelper.isWebApp ? '-p:PublishProfile=DefaultContainer' : '/t:PublishContainer';
+        const [runtimeOs, runtimeArch] = this.getOsAndArch(buildDefinition);
+        const [imageName, imageTag] = this.getImageNameAndTag(buildDefinition);
+
+        const sdkBuildCommand = `dotnet publish --os ${runtimeOs} --arch ${runtimeArch} ${publishFlag} -c Release -p:ContainerImageName=${imageName} -p:ContainerImageTag=${imageTag}`;

```

Comment:
The build configuration should probably default to Debug instead of Release. It would make sense to have a discussion about how/if we would want to make the property configurable (without breaking existing behavior).

```suggestion
        const sdkBuildCommand = `dotnet publish --os ${runtimeOs} --arch ${runtimeArch} ${publishFlag} -c Debug -p:ContainerImageName=${imageName} -p:ContainerImageTag=${imageTag}`;
```

Reply:
That makes sense, I'll change it to debug for now and I'll bring this up at standup tomorrow.